### ISSUE TITLE
Add support to add new custom command just like django manage.py custom commands

### ISSUE THREAD

**Issue by cyantarek** [beego/bee#534]
Hello, the beego framework and bee tool is excellent. But missing some features.

Such as, I want a functionality to add custom bee command very much like django.

You can see how Django implements it from [here](https://docs.djangoproject.com/en/2.0/howto/custom-management-commands/) as a reference.

Everybody will appreciate it.
Thanks.

**Comment by guhan121** [comment]
I don't think this is necessary, go is different from Python, go must be compiled before it can be executed, with such an event we can make a better tool instead of extending bee
